irohad/model/execution/impl/command_executor.cpp

```diff
@@ -482,16 +512,24 @@ namespace iroha {
       account.json_data = "{}";
```

Comment:
Looks very suspicious

Reply:
It just creates an empty json object. Sadly we do not have any better way of doing that in old model